models/TradingBot.py

```diff
@@ -77,24 +85,34 @@ def perform_quicktrade(self, update: Update, context: CallbackContext) -> int:
                 else:
                     instrument_type = trade_elements[3].lower()
 
-                instrument = Instrument().get_quick_isin(search, instrument_type)
+                instrument_list = client.market_data.instruments.get(search=search, type=instrument_type).results
+                print(instrument_list)
```

Comment:
Thanks for pointing this out. That's a print left over from debugging the bot, I'll use your suggested print since it is useful for seeing when the search can't find any instruments.

Reply:
Cool! Please look through the other prints as well :)